### ISSUE TITLE
Request:Enabling Web Access for Documentation Search in "gpt-engineer" 

### ISSUE THREAD

**Issue by zikass94** [AntonOsika/gpt-engineer#46]
I'm wondering if it's possible to enable the "gpt-engineer" to access the web for searching documentations.

**Comment by Mayorc1978** [comment]
> I'm wondering if it's possible to enable the "gpt-engineer" to access the web for searching documentations.

Definitely, supports for embeddings, pdfs, and web search is a must, else it would be very hard to get something working with updated frameworks apis.

**Comment by Mayorc1978** [comment]
> Doesn't the latest gpt model support functions now?
> 
> It should be possible in theory to implement web access.

Supporting paid-only features sounds like a poor choice to me.

**Comment by w3bdesign** [comment]
> > Doesn't the latest gpt model support functions now?
> > It should be possible in theory to implement web access.
> 
> Supporting paid-only features sounds like a poor choice to me.

I don't think it is a paid feature.

**Comment by patillacode** [comment]
Hi!

Yes, almost anything is possible these days, the latest feature provided by openai (not a paid feature AFAIK) are functions.
That would potentially allow for us/you to create a "search google" function.

See #58 for a first approach.
